Darnton.Blazor.Leaflet/LeafletMap/MarkerOptions.cs

```diff
@@ -1,9 +1,69 @@
-﻿namespace Darnton.Blazor.Leaflet.LeafletMap
+﻿using System.Text.Json.Serialization;
+
+namespace Darnton.Blazor.Leaflet.LeafletMap
 {
     /// <summary>
     /// The options used when creating a <see cref="Marker"/>.
     /// </summary>
     public class MarkerOptions
     {
+        /// <summary>
+        /// Whether the marker can be tabbed to with a keyboard and clicked by pressing enter.
+        /// </summary>
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public bool? Keyboard { get; set; }
+
+        /// <summary>
+        /// Text for the browser tooltip that appear on marker hover (no tooltip by default).
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// Text for the alt attribute of the icon image (useful for accessibility).
+        /// </summary>
+        public string Alt { get; set; }
+
+        /// <summary>
+        /// By default, marker images zIndex is set automatically based on its latitude. Use this option if you want to
+        /// put the marker on top of all others (or below), specifying a high value like 1000 (or high negative value, respectively)
+        /// </summary>
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
```

Comment:
This attribute and condition mean that if the value is null, i.e. unset by the instantiater, it will not be serialized. I did this to allow the Leaflet.js defaults to be utilized, and not require all options to be set or have the defaults coded in this class.